ui/src/utils/defaults.js

```diff
@@ -1,12 +1,11 @@
-  // GENERATED FROM /Users/katesills/code/dapp-simple-exchange/api/deploy.js
-  export default {
-  "INSTANCE_BOARD_ID": "5007165",
-  "INVITATION_BRAND_BOARD_ID": "16679794",
-  "ASSET_BRAND_BOARD_ID": "7279951",
-  "PRICE_BRAND_BOARD_ID": "3715714",
-  "BRIDGE_URL": "http://127.0.0.1:8000",
-  "API_URL": "http://127.0.0.1:8000",
-  "CONTRACT_NAME": "simple-exchange",
-  "INSTALLATION_BOARD_ID": "14561541"
+// GENERATED FROM /Users/katesills/code/dapp-simple-exchange/api/deploy.js
+export default {
+  INSTANCE_BOARD_ID: '605437304',
+  INVITATION_BRAND_BOARD_ID: '1667979430',
+  ASSET_BRAND_BOARD_ID: '262188032',
+  PRICE_BRAND_BOARD_ID: '1202180815',
```

Comment:
Should this generated file be included?

Reply:
Hmm, good point. I think not.

Reply:
Ah, I didn't include it in this PR. I think the issue is known generally, but I'll make an issue about .gitignoring `defaults.js` while still ensuring the directory exists for everyone

Reply:
https://github.com/Agoric/dapp-simple-exchange/issues/29